package.json

```diff
@@ -54,6 +54,6 @@
     "karma-coverage": "^2.0.3",
     "karma-jasmine": "^4.0.0",
     "karma-jasmine-html-reporter": "^1.7.0",
-    "typescript": "^4.6.4"
+    "typescript": "4.8"
```

Comment:
update version.. 
instead of 4.8, use ^ to denote that it can be any major version 4